include/call.hpp

```diff
@@ -74,8 +74,8 @@ class call : virtual public task, virtual public listener, public virtual socket
 
     virtual ~call();
 
-    virtual bool process_incoming(char * msg, struct sockaddr_storage *src = NULL);
-    virtual bool  process_twinSippCom(char * msg);
+    virtual bool process_incoming(char* msg, struct sockaddr_storage* src = NULL, SIPpSocket* sock = NULL);
+    virtual bool process_twinSippCom(char * msg);
```

Comment:
Is there a reason why you've removed the white space between the type declaration and the pointer specifier on line 77 but haven't done so here?